### DISCUSSION TITLE
How to access pihole db on different machine

### DISCUSSION THREAD

**Discussion by dilipprasad** [netalertx/NetAlertX#273] [General]
Hi,

I am trying to complete the installation, however i have setup pihole on a raspberry pi 0  and the the PiAlert is in a different machine that is raspberry pi 4. How do i setup the path for the below 2 db ?

PIHOLE_ACTIVE: You need to map :/etc/pihole/pihole-FTL.db in the docker-compose.yml file if you enable this setting.
DHCP_ACTIVE : You need to map :/etc/pihole/dhcp.leases in the docker-compose.yml file if you enable this setting.

**Comment by jokob-sk** [comment]
Hey,

You have to solve this on the server level. It's not something that is supported in PiAlert.

You need to physically map these files in the file system in the docker compose file. If you have 2 different servers you could synchronize these files with a 3rd party app, such as syncthing

j

**Comment by dilipprasad** [comment]
Great, thank you
